communitiesuk/funding-service-design-post-award-docker-runner
# Add celery auto-reload

### Change description
Add celery auto-reload.

New ENV to enable it `CELERY_AUTO_REFRESH`. If exists and `true`, it auto restarts the celery worker.

### How to test
- Pull this branch
- Pull data-store branch from [this PR](https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/750)
- Run docker runner
- Download a find report
- Make a change to any .py file in the project (I added a print in the download async function)
- You will not see docker-runner say that celery was restarted
- OR if you added a `print` you will not see it printed on download
- Stop docker runner
- Set inside the docker runner CELERY_AUTO_REFRESH to `true`
- Start docker runner
- Make a change to any .py file in the project (I added a print in the download async function)
- You will not see docker-runner say that celery was restarted
- OR if you added a `print` you will not see it printed on download
